static/src/js/components/KeyboardShortcutsModal/KeyboardShortcutsModal.scss

```diff
@@ -0,0 +1,3 @@
+.row {
```

Comment:
Remove this file after replacing the custom class name with the boostrap class name.